search_commands.go

```diff
@@ -293,8 +295,9 @@ type FTSearchSortBy struct {
 // More information about the options can be found
 // in the documentation for FT.SEARCH https://redis.io/docs/latest/commands/ft.search/
 type FTSearchOptions struct {
-	NoContent    bool
-	Verbatim     bool
+	NoContent bool
+	Verbatim  bool
+	// Deprecated: NoStopWords is deprecated since Redis 8.0
 	NoStopWords  bool
```

Comment:
Looking at the changelog for redis 8, I did not see deprecation for `NOSTOPWORDS`. @ofekshenawa can you point me to where you found that, maybe I am missing it.